src/main/java/org/apereo/openequella/adminconsole/launcher/ClientLauncher.java

```diff
@@ -220,7 +220,7 @@ public Object doInBackground() throws Exception {
           loadingDialog.setVisible(false);
 
           jarService.executeJar(jarName, "com.tle.admin.boot.Bootstrap", "-Djnlp.ENDPOINT=" + url,
-              "-Dplugin.cache.dir=" + StorageService.getCacheFolder(uuid, "cache"));
+              "-Dplugin.cache.dir=" + StorageService.getCacheFolder(uuid, "cache"), "-DSERVER_NAME="+serverProfile.getName());
```

Comment:
The formatting seems to be not working.  Should have spaces around the +
Not enough for me to care about stopping the pull request though :)

Reply:
I forgot we don't have formatting tool for this project.